### DISCUSSION TITLE
i20230521 - Does not boot on RPi 3B and RPi 3B+

### DISCUSSION THREAD

**Discussion by 59iosl30** [rern/rAudio#1370] [Bug Reports]
Hello!

I wrote the image file [rAudio-64bit-20230521.img.xz](https://github.com/rern/rAudio/releases/tag/i20230521) to the sdcard and tried to boot it on a RPi 3B and RPi 3B+. The boot logo "+R" remains on the screen and the Raspberry Pi activity led does not stop blinking.

Why does the image not boot or am I doing something wrong?

Kind regards,
Robert

**Comment by kw-sudo** [comment]
Hi Robert,

yesterday I've started a clean install from the scratch on my RasPi 3B+ with that image (BTW, it's the only newer version that works for me without issues). Did you checked your SD-card? Have you alternatively unpacked the *.xz image and flashed it as *.img file?

Oliver

**Reply by 59iosl30** [reply]
Hello Oliver!

Yes, both sd cards are okay. I unpacked the image and then I flashed it.

Kind regards.
Robert

**Comment by markblinkhorn** [comment]
I use Rufus (PC version) to write the SD card direct from the .xz image file (no need to decompress) and have not had any problems.

regards
Mark

(edited to fix the obvious typo)

**Reply by kw-sudo** [reply]
I use balenaEtcher (macOS) and there‘s also no need to decompress, but it‘s faster.

**Comment by ceejayemm** [comment]
Try connecting a web browser to rAudio.local or the IP number of the machine as the app is web based and meant to be accessed via a web browser.

Chris

**Comment by R101G** [comment]
It sounds like it's booting, but getting stuck in a library scan, or some other initial task. What media do you have connected?

**Reply by 59iosl30** [reply]
Hello!

Thanks for all hints!

@R101G: I have connected a network cable and now I get the default screen and can also connect to the web interface.

Why does the boot time take very long until I get the default screen on the HDMI output when there is no network cable connected for offline use ?

Btw: I used the linux core command line utils dd and unxz
$ sudo unxz -cd rAudio-64bit-20230521.img.xz |sudo dd of=/dev/mmcblk0

**Caution**:
The above command overwrites all data in the sd card reader.

Kind regards,
Robert

**Comment by 59iosl30** [comment]
> Why does the boot time take very long until I get the default screen on the HDMI output when there is no network cable connected for offline use ?

It looks like the startup script "/srv/http/bash/startup.sh" is causing the boot problem.

$ sudo systemd-analyze plot >plot.svg
startup.service (1 min 42.686s) :-(

A quick but dirty workaround is to comment out all WIFI (WLAN) relevant parts.

The system boots fast now:
$ sudo systemd-analyze
startup finished in 4.049s (kernel) + 6.983s (userspace) = 11.032s 
graphical.target reached after 6.982s in userspace.

I disabled all wireless devices:
/boot/config.txt:
dtoverlay=pi3-disable-wifi
dtoverlay=pi3-disable-bt
or on RPi4:
dtoverlay=disable-wifi
dtoverlay=disable-bt

With kind regards,
Robert
